serverless/messaging/api-cli/connect-aws-cli.mdx

```diff
@@ -1,28 +1,28 @@
 ---
 meta:
-  title: Connecting your SQS/SNS namespace to the AWS-CLI
-  description: This page explains how to connect your Scaleway Messaging and Queuing SQS/SNS namespace to the AWS-CLI
+  title: Connecting Messaging and Queing SQS or SNS to the AWS-CLI
+  description: This page explains how to connect Scaleway Messaging and Queing SQS or SNS to the AWS-CLI
 content:
-  h1: Connecting your SQS/SNS namespace to the AWS-CLI
-  paragraph: This page explains how to connect your Scaleway Messaging and Queuing SQS/SNS namespace to the AWS-CLI
+  h1: Connecting Messaging and Queing SQS or SNS to the AWS-CLI
+  paragraph: This page explains how to connect Scaleway Messaging and Queing SQS or SNS to the AWS-CLI
 tags: messaging sns sqs aws-cli cli aws sdk python boto
 categories:
   - messaging
 dates:
-  validation: 2023-07-10
+  validation: 2023-09-13
   posted: 2023-01-04
 ---
 
-The AWS-CLI is an open-source tool built on top of the AWS SDK for Python (Boto) that provides commands for interacting with AWS services. With minimal configuration, you can start using the AWS-CLI with your Scaleway Messaging and Queuing SQS/SNS namespace. This allows you to create, list and manage your queues and topics, send messages and much more, all from your command line.
+The AWS-CLI is an open-source tool built on top of the AWS SDK for Python (Boto) that provides commands for interacting with AWS services. With minimal configuration, you can start using the AWS-CLI with Scaleway Messaging and Queuing SQS and/or SNS. This allows you to create, list and manage your queues and topics, send messages and much more, all from your command line.
 
-This guide shows you how to install the AWS-CLI and configure it for connection to your Scaleway Messaging and Queuing SQS/SNS namespace.
+This guide shows you how to install the AWS-CLI and configure it for connection to Scaleway Messaging and Queuing SQS or SNS.
```

Comment:
You're probably more fluent in English than me but `for connecting to Scaleway Messaging ...` sounds more correct to me.